vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -51,37 +50,35 @@ A customized image can include:
 * New APIs 
 * Legacy Entando plugins
 * New database tables  
-* Other extensions to the **entando-core** 
+* Other extensions to the entando-core
 
-It is recommended that most extensions to the Entando Platform occur in microservices. However, legacy 
+It is recommended that microservices are used for most extensions to the Entando Platform. However, legacy 
```

Comment:
this should be tightened up to clarify whether the extension IS a MS or if an extension facilitates an MS

Reply:
it's neither, it's just saying to use ms to extend the platform